firmware/config/chconf_common.h

```diff
@@ -12,6 +12,12 @@
 #include "obd_error_codes.h"
 #endif /* _FROM_ASM_ */
 
+
+#ifndef ENABLE_PERF_TRACE
+// looks like this value could not be defined in efifeatures.h - please define either externally or just change the value here
+ #define ENABLE_PERF_TRACE FALSE
```

Comment:
@rusefi Do we have a preference about the default here? Normal users probably don't have a need to enable it, and there is a small (but extant) performance impact to leaving it on.

Reply:
My preference is TRUE
I would hope that with our gigantic CPU we shall afford some monitoring features